doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -197,6 +237,40 @@ with a ``.ipynb`` file.
 
 .. _MyST-NB: https://myst-nb.readthedocs.io/
 
+.. _interactive-docs:
+
+Interactive examples in docstrings
+----------------------------------
+
+Examples within docstrings can be made interactive using the `jupyterlite-sphinx extension <https://jupyterlite-sphinx.readthedocs.io/en/latest/>`__,
```

Comment:
```suggestion
Examples within docstrings can be made interactive using the `jupyterlite-sphinx extension <https://jupyterlite-sphinx.readthedocs.io/en/stable/>`__,
```

Nit: probably linking to the stable version is better

Reply:
In that case it should also probably be better to point the jupyterlite docs page default to be stable instead of latest

Reply:
Thanks @melissawm, that's a good point. I have admin permissions there, so I modified that in the RTD settings just now. I think the change should reflect within a few hours/days.